suites/squid/common/sanity/rgw.yaml

```diff
@@ -24,63 +24,26 @@ tests:
-        - test:
-            name: Test M buckets with N objects
-            desc: test to create "M" no of buckets and "N" no of objects
-            polarion-id: CEPH-9789
-            module: sanity_rgw.py
-            config:
-              script-name: test_Mbuckets_with_Nobjects.py
-              config-file-name: test_Mbuckets_with_Nobjects.yaml
-              install_common: false
-              run-on-rgw: true
-
-        - test:
-            name: Test delete using M buckets with N objects
-            desc: test to create "M" no of buckets and "N" no of objects with delete
-            polarion-id: CEPH-14237
-            module: sanity_rgw.py
-            config:
-              script-name: test_Mbuckets_with_Nobjects.py
-              config-file-name: test_Mbuckets_with_Nobjects_delete.yaml
-              install_common: false
-              run-on-rgw: true
-
-        - test:
-            name: Test download with M buckets with N objects
-            desc: test to create "M" no of buckets and "N" no of objects with download
-            polarion-id: CEPH-14237
-            module: sanity_rgw.py
-            config:
-              script-name: test_Mbuckets_with_Nobjects.py
-              config-file-name: test_Mbuckets_with_Nobjects_download.yaml
-              install_common: false
-              run-on-haproxy: true
-
-        - test:
-            name: Test multipart upload of M buckets with N objects
-            desc: test to create "M" no of buckets and "N" no of objects with multipart upload
-            polarion-id: CEPH-9801
-            module: sanity_rgw.py
-            config:
-              script-name: test_Mbuckets_with_Nobjects.py
-              config-file-name: test_Mbuckets_with_Nobjects_multipart.yaml
-              install_common: false
-              run-on-rgw: true
+      name: Test download with M buckets with N objects
+      desc: test to create "M" no of buckets and "N" no of objects with download
+      polarion-id: CEPH-14237
+      module: sanity_rgw.py
+      config:
+        script-name: test_Mbuckets_with_Nobjects.py
+        config-file-name: test_Mbuckets_with_Nobjects_download.yaml
+        install_common: false
+        run-on-haproxy: true
 
-        - test:
-            name: test_bi_purge for a bucket
-            desc: test bi_purge should not error
-            polarion-id: CEPH-83575234
-            module: sanity_rgw.py
-            config:
-              install_common: false
-              script-name: test_Mbuckets_with_Nobjects.py
-              config-file-name: test_bi_purge.yaml
+  - test:
+      name: Test multipart upload of M buckets with N objects
+      desc: test to create "M" no of buckets and "N" no of objects with multipart upload
+      polarion-id: CEPH-9801
+      module: sanity_rgw.py
+      config:
+        script-name: test_Mbuckets_with_Nobjects.py
+        config-file-name: test_Mbuckets_with_Nobjects_multipart.yaml
+        install_common: false
```

Comment:
@yuva-gayam , since its not parallel run we can remove config : install_common
and also can you please check on run-on-rgw, that do we need this?